### DISCUSSION TITLE
Disable auto-complete pop-ups without losing other functionality?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [coder/code-server#6538] [Q&A] [answered]
I'm not sure when this changed, but for some reason for the past few days/weeks, I keep getting auto-complete popups then when I hit enter as I'm typing, replace the correct text with sometimes incorrect text from the pop-up. Is there any way to turn these off but still retain the rest of the functionality? I'm not a coder, I use code-server to alter YAML and JSON config files quite frequently, so I haven't changed any default settings other than the theme. This is driving me nuts though, and I can't seem to find the setting to turn it off.
![Screen Shot 2023-11-16 at 4 49 18 PM](https://github.com/coder/code-server/assets/60272670/65911d72-7c67-42f4-a4fd-41ca4b0202bc)

**Comment by code-asher** [comment, marked as answer]
I have not tried this myself but maybe this helps: https://www.reddit.com/r/vscode/comments/ayvd0p/how_to_stop_autocomplete_with_enter/

Supposedly setting `editor.acceptSuggestionOnEnter` to `off` works.

**Reply by SawkeeReemo** [reply]
Incredible, thank you! And on that reddit link, there was a comment that showed how to find this in the UI's settings:
https://imgur.com/EAkDgno

In case this imgur link dies, here is a screen shot from my own system as of today 11/18/2023:
![Screen Shot 2023-11-17 at 1 56 18 PM](https://github.com/coder/code-server/assets/60272670/5f00dcce-b525-4707-9203-0caeef5987ea)
